templates/index.jade

```diff
@@ -51,7 +51,9 @@ link(href='/css/others_for_adapted_pages.css', rel='stylesheet')
         // Математика. Просто
         +course(locals.courses.math_simple)
         // Соціальне підприємництво та конкурентоспроможність
-        +course(locals.courses.soc_ent)
+        // +course(locals.courses.soc_ent)
+         // ОСНОВИ ВЕБ-РОЗРОБКИ (HTML, CSS, JAVASCRIPT)
```

Comment:
@Miracle-Aligner be careful with spaces/tabs in jade. Does it make any side effects?

Reply:
@Enzewon 
this part, as a comment, makes no side effects. 
stage.ed-era.com looks just fine
but fixed anyways, tnx!

btw Vlad asked to wait with merge till 11 pm.